src/test/java/uk/ac/ebi/eva/pipeline/jobs/PopulationStatisticsJobTest.java

```diff
@@ -68,8 +70,11 @@
     @Autowired
     private JobLauncherTestUtils jobLauncherTestUtils;
 
+    private static String opencgaHome = GenotypedVcfJobTestUtils.getDefaultOpencgaHome();
+
     @Test
     public void fullPopulationStatisticsJob() throws Exception {
+        Config.setOpenCGAHome(opencgaHome);
```

Comment:
Shall we take the chance and move all these statements to a `setUp` like in other classes?